src/protocol.cpp

```diff
@@ -304,13 +305,32 @@ CONNECTION LESS MESSAGES
     where an odd number of clients is connected, there will be four unused
     upper bits in the final byte, containing 0xF (which is out of range)
 
-    the server may compute them message when any client has used
+    the server may compute the message when any client has used
     PROTMESSID_CLM_REQ_CHANNEL_LEVEL_LIST to opt in
 
-    the server may issue to message only to a client that has used
+    the server should issue the message only to a client that has used
     PROTMESSID_CLM_REQ_CHANNEL_LEVEL_LIST to opt in
 
 
+- PROTMESSID_CLM_REGISTER_SERVER_RESP: result of registration request
+
+    +---------------+
+    | 1 byte status |
+    +---------------+
+
+    - "status":
```

Comment:
As far as I understood, you are transmitting the enum now and not a single bit, right? Then you should update the documentation here, too.

Reply:
I will merge your changes now. Please fix this in another pull request later on.